metadata.json

```diff
@@ -73,6 +73,12 @@
         "12.04",
         "14.04"
       ]
+    },
+    {
+      "operatingsystem": "Gentoo",
+      "operatingsystemrelease": [
+        "1.0"
```

Comment:
Since Gentoo isn't supported for the search functionality within Puppet Forge (and this is, from my understanding, the only thing metadata.json is used for) this should be fine! :)